docs/distributed/README.md

```diff
@@ -1,74 +1,85 @@
 
-## 2. Run distributed Minio
+### <a name="install-minio-server"></a>1. Install Minio Server
 
-To start a distributed Minio instance, you just need to pass drive locations as parameters to the minio server command. Then, you’ll need to run the same command on all the participating nodes.
+Install Minio Server using the instructions in the [Minio Quickstart Guide](https://docs.minio.io/docs/minio-quickstart-guide).
 
-*Note*
+### <a name="run-distributed-minio"></a>2. Run Distributed Minio
 
-- All the nodes running distributed Minio need to have same access key and secret key for the nodes to connect. To achieve this, it is **mandatory** to export access key and secret key as environment variables, `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`, on all the nodes before executing Minio server command.
-- All the nodes running distributed Minio need to be in a homogeneous environment, i.e. same operating system, same number of disks and same interconnects.
-- `MINIO_DOMAIN` environment variable should be defined and exported if domain is needed to be set.
-- Minio distributed mode requires fresh directories. If required, the drives can be shared with other applications. You can do this by using a sub-directory exclusive to minio. For example, if you have mounted your volume under `/export`, pass `/export/data` as arguments to Minio server.
-- The IP addresses and drive paths below are for demonstration purposes only, you need to replace these with the actual IP addresses and drive paths/folders.
-- Servers running distributed Minio instances should be less than 3 seconds apart. You can use [NTP](http://www.ntp.org/) as a best practice to ensure consistent times across servers.
-- Running Distributed Minio on Windows is experimental as of now. Please proceed with caution.
+To start a distributed Minio instance:
+1. Pass the hard drive locations as parameters to the `minio server` command.
+2. Re-run the same command on all of the participating nodes.
+
+**Note:**
+* All nodes running distributed Minio must have the same access key and secret key for the nodes to connect. This requires that the access key and secret key be exported as the following environment variables on all nodes before executing the `minio server` command: `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`.
+* All nodes running distributed Minio must be in a homogeneous environment (meaning they have the same operating system, number of drives, and interconnects).
+* If the domain must be set, the `MINIO_DOMAIN` environment variable must be defined and exported.
+* Minio distributed mode requires directories that are exclusive to Minio. For example, if a volume is mounted under `/export`, pass `/export/data` as an argument to the `minio server` command.
+**Note:** The drives can be shared with other applications if required.
```

Comment:
"The drives can be shared with other applications" - This isn't clear, we should either point to a way to do this or remove it as it is too ambiguous.  @deekoder thoughts?